src/tdl/queue/implementation_runner_config.py

```diff
@@ -1,4 +1,4 @@
-from tdl.audit.stdout_audit_stream import StdoutAuditStream
+from src.tdl.audit.stdout_audit_stream import StdoutAuditStream
```

Comment:
We should not have "src" in front.
You should explore ways of loading the "src" folder with Python3

Reply:
This is the last thing to sort and we should be able to release the client.